### PR TITLE
Release/1.0.0

### PR DESCRIPTION
**Pull request by iSeremet-Reloadly**
First release version.
1.0.0

### CODE REVIEW

File: .github/workflows/npm-publish-beta.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+name: Build and Publish NPM Package
+
+on:
+    push:
+        branches:
+            - develop
```
Comment (farouk-reloadly) [inline .github/workflows/npm-publish-beta.yml:6]: Just to confirm, @JohnnyTMD  won't you have to include the master branch in the list of branches here?

Comment (iSeremet-Reloadly) [reply .github/workflows/npm-publish-beta.yml:6]: hmm, maybe
the thing is that I had no way to test that previously

File: .github/workflows/npm-publish-beta.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            - name: Build
+              run: npm ci && npm run build:prod
+
+            - name: Publish to npm (beta version)
+              run: npm run npm:publish:beta
+              env:
+                  NODE_AUTH_TOKEN: ${{secrets.NPM_AUTH_TOKEN}}
+    deploy-release:
+        runs-on: ubuntu-latest
+        if: github.ref == 'refs/heads/master'
```
Comment (farouk-reloadly) [inline .github/workflows/npm-publish-beta.yml:34]: Also, the default branch is `main` but you used `master`, would that change anything?

File: .github/workflows/npm-publish-beta.yml (hunk 1 above)
Comment (iSeremet-Reloadly) [reply .github/workflows/npm-publish-beta.yml:6]: I will merge this one and if the pipeline will not be triggered I will add a hotfix

Comment (farouk-reloadly) [reply .github/workflows/npm-publish-beta.yml:6]: Alright, that sounds good too

Comment (iSeremet-Reloadly) [reply .github/workflows/npm-publish-beta.yml:6]: I have added the fixes now,

**Review by salimzakari-reloadly — APPROVED** [review]

**Pull request merged**